consensus/spos/bls/v2/subroundEndRound.go

```diff
@@ -332,38 +344,40 @@ func (sr *subroundEndRound) finalizeConfirmedBlock() bool {
 	return true
 }
 
-func (sr *subroundEndRound) sendProof() {
+func (sr *subroundEndRound) sendProof() error {
 	if !sr.shouldSendProof() {
-		return
+		return errShouldNotSendProof
```

Comment:
i think it would be cleaner to return `bool, error` to specify if the proof was sent

Reply:
added